superset/config.py

```diff
@@ -612,6 +612,15 @@ class CeleryConfig(object):
     'presto': 'PrestoDBSQLValidator',
 }
 
+# Do you want Talisman enabled?
+TALISMAN_ENABLED = True
+# If you want Talisman, how do you want it configured??
+TALISMAN_CONFIG = {
+    'content_security_policy': None,
+    'force_https': True,
```

Comment:
Per the Flask-Talisman [documentation](https://github.com/GoogleCloudPlatform/flask-talisman) the `force_https` default is `True` and thus it seems we could nix this line.

Reply:
Yep, that's where I got that key. Wanted it explicit here so that others will know that it might be causing trouble (as it was for me).

Reply:
Personally I find it a little confusing to include only a subset of the defaults. Maybe there is merit in being more explicit in the comment to include i) a link to the documentation which lists the defaults, and ii) mentioning that this really isn't the entire configuration but rather an augmentation of the explicitly mentioned parameters.

Reply:
@craig-rueda would you be able to follow up on this comment in a future PR?

Reply:
Yes, of course